src/i18n/en.js

```diff
@@ -27,6 +27,19 @@ const urlSiteContact = `${urlSite}/en/contact/`;
 const urlControlPanelMain = `${urlControlPanel}/ControlPanel`;
 
 export default {
+  campaign_history: {
```

Comment:
How about renaminf this into `subscriber`? because we will use this from the grid as well.
```suggestion
  subscriber: {
```

Reply:
ready